.github/workflows/testing.yml

```diff
@@ -109,6 +124,10 @@ jobs:
         options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3 -e MYSQL_ROOT_PASSWORD=root -e MYSQL_DATABASE=wp_cli_test --entrypoint sh mysql:${{ matrix.mysql }} -c "exec docker-entrypoint.sh mysqld --default-authentication-plugin=mysql_native_password"
 
     steps:
+
+      - name: Test connect to MySQL
+        run: mysql -uroot -proot -h127.0.0.1 -P${{ job.services.mysql.ports[3306] }} -e "SHOW DATABASES"
```

Comment:
this canary can be removed I guess. I mostly wanted to shed all the potential that the behat and testing side was messing up path or not receiving environment variables.